pkg/devfile/adapters/common/command.go

```diff
@@ -107,6 +107,19 @@ func validateAction(data data.DevfileData, action common.DevfileCommandAction) (
 	return
 }
 
+// GetInitCommand iterates through the components in the devfile and returns the init command
+func GetInitCommand(data data.DevfileData, devfileInitCmd string) (initCommand common.DevfileCommand, err error) {
+	if devfileInitCmd != "" {
+		// a init command was specified so if it is not found then it is an error
+		initCommand, err = getCommand(data, devfileInitCmd, true)
```

Comment:
you can directly return the `getCommand(...)` instead of assigning it to anything.
And once you do that golint will complain about not having an `else` clause as if `returns`. so please do that change too.

Reply:
also if possible can we change this for  `GetBuildCommand` and `GetRunCommand`, those are not part of this PR but it would be great to have some consistency :slightly_smiling_face: 
ref - https://github.com/openshift/odo/blob/51134ae769c00463a584df6c79887b01572d43c2/pkg/devfile/adapters/common/command.go#L124